examples/simple/main.tf

```diff
@@ -0,0 +1,71 @@
+terraform {
+  required_version = ">= 0.11.2"
+
+  backend "s3" {}
+}
+
+variable "aws_assume_role_arn" {}
+
+provider "aws" {
+  assume_role {
+    role_arn = "${var.aws_assume_role_arn}"
+  }
+}
+
+variable "namespace" {}
+
+variable "name" {}
+
+variable "stage" {}
+
+variable "region" {}
+
+variable "availability_zones" {
+  type = "list"
+}
+
+variable "zone_id" {}
+
+module "vpc" {
+  source    = "git::https://github.com/cloudposse/terraform-aws-vpc.git?ref=tags/0.3.3"
+  namespace = "eg"
+  stage     = "testing"
+  name      = "redis"
+}
+
+module "subnets" {
+  source             = "git::https://github.com/cloudposse/terraform-aws-dynamic-subnets.git?ref=tags/0.3.5"
+  namespace          = "eg"
```

Comment:
"${var.namespace}"?

should not mix the hardcoded values with variables.
we have `namespace`, `stage` and `name` vars.
and all other attributes are set from vars.